app/views/layouts/application.html.erb

```diff
@@ -13,7 +13,8 @@
   </head>
 
   <body>
-    <%= render 'components/navbar' %>
+    <%= render 'shared/navbar' %>
```

Comment:
It needs to be components/navbar otherwise it will not render Francesca's navbar.